app/javascript/components/mentoring/discussion/Guidance.tsx

```diff
@@ -45,21 +48,178 @@ export const Guidance = (): JSX.Element => {
   )
 
   return (
-    <div>
+    <>
+      <Accordion id="notes" isOpen={isOpen('notes')} onClick={handleClick}>
+        <Accordion.Header>Mentor notes</Accordion.Header>
+        <Accordion.Panel>
+          <div className="c-textual-content --small">
```

Comment:
If notes is `present?` then this should show. There will also be a CTA to improve the notes. 

If notes is not present, then we'll show a (different) CTA to contribute instead.